### ISSUE TITLE
Include "Planned" as a support indicator

### ISSUE THREAD

**Issue by msub2** [limbofeather/worlds#2]
Many projects might have one or two aspects listed on their roadmap as a future feature, which I think would be worth distinguishing from a project that doesn't mention it at all. Sougen, for example, has stated that VR/WebXR support is planned down the road.

**Comment by limbofeather** [comment]
Nice suggestion @msub2 - if we're going to have "Planned", then should we also have the planned data on when something would land?

**Comment by msub2** [comment]
Yeah, I think ideally you would link "Planned" to the relevant page or info source stating its on their roadmap/planned for the future

**Comment by limbofeather** [comment]
I don't have any objections to adding this as a support indicator, I'm open to adding the indicator for Sougen as long as there's some sort of link to the fact that it's on their roadmap.
